libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -405,6 +406,7 @@ method publish*(g: GossipSub,
       trace "publishing on topic", name = topic
       g.mcache.put(msg)
       sent.add(g.peers[p].send(@[RPCMsg(messages: @[msg])]))
+    # TODO FIXME what to do with erros here??
```

Comment:
@dryajov Should we just leave `allFutures` and let it throw here? (repeating the question few times around)

Reply:
same as previous